lib/fastlane/plugin/wpmreleasetoolkit/helper/promo_screenshots_helper.rb

```diff
@@ -233,9 +235,7 @@ def draw_text_to_canvas(canvas, text, width, height, x_position, y_position, fon
         begin
           tempTextFile = Tempfile.new()
 
-          command = "bundle exec drawText html=\"#{text}\" maxWidth=#{width} maxHeight=#{height} output=#{tempTextFile.path} fontSize=#{font_size} stylesheet=\"#{stylesheet_path}\" alignment=\"#{position}\""
-
-          UI.crash!('Unable to draw text') unless system(command)
+          sh('drawText', "html=\"#{text}\"", "maxWidth=#{width}", "maxHeight=#{height}", "output=#{tempTextFile.path}", "fontSize=#{font_size}", "stylesheet=\"#{stylesheet_path}\"", "alignment=\"#{position}\"")
```

Comment:
Even if this is unrelated to this PR per se (as the issue was apparently already present before that diff), would be nice to ensure the output path is quoted to avoid issues with spaces in path. I know it's unlikely because temp files usually reside in `/tmp` and don't usually have spaces in their path, but still better safe than sorry 😃 

```suggestion
          sh('drawText', "html=\"#{text}\"", "maxWidth=#{width}", "maxHeight=#{height}", "output=\"#{tempTextFile.path}\"", "fontSize=#{font_size}", "stylesheet=\"#{stylesheet_path}\"", "alignment=\"#{position}\"")
```

Reply:
Actually, that was an incorrect suggestion here, because we now already pass parameters separately to `sh` — as opposed to passing a single, long string with spaces and leave to the shell interpreter to split parameters based on spaces and quotes — so `sh(x, y, z)` already passes the separate `y` and`z` parameters to the `x` executable directly without any shell interpreter step to split them, and thus without the need to even quote/escape them.

The escaping/quoting is only needed if we pass a single string to `sh`/`system` and let the shell interpreter to the split of the parameters from that long, single string. So here the right solution would have been to not only keep the `tempTextFile.path` unquoted as before, but also to remove the various `\"` quotes from other parameters like `html=`, `stylesheet=` and `alignment=`.

Kudos to @rachelmcr for spotting this! [ref: pdnsEh-hU-p2#comment-699]